### ISSUE TITLE
h5py Dockerfile

### ISSUE THREAD

**Issue by bhack** [tensorflow/tensorflow#18279]
h5py is a quite hard dependency but it is not in all the flavors of TF Dockerfile.
I.e. https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/keras/_impl/keras/applications/vgg16.py#L50-L51

See https://github.com/keras-team/keras/pull/9830
/cc @fchollet

**Comment by tensorflowbutler** [comment]
Nagging Assignee @aselle: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by aselle** [comment]
@angersson, do you know who is working on docker right now?

**Comment by bhack** [comment]
@av8ramit?

**Comment by bhack** [comment]
See also https://github.com/tensorflow/tensorflow/pull/11360

**Comment by bhack** [comment]
@caisq Do you know why it was not introduced in all the Dockerfile on #11360 review process.

**Comment by caisq** [comment]
@bhack @av8ramit I'm fine with adding h5py to the devel Docker images if users require it.

**Comment by angersson** [comment]
I'll put together a change to add the dependencies to the dockerfiles.

**Comment by av8ramit** [comment]
https://github.com/tensorflow/tensorflow/pull/18868

**Comment by angersson** [comment]
Resolved by #18868. Thanks Amit!
